src/app/zap-templates/zcl/data-model/chip/smoke-co-alarm-cluster.xml

```diff
@@ -0,0 +1,153 @@
+    </event>
+    <event side="server" code="0x06" name="AlarmMuted" priority="info" optional="true">
+      <description>This event SHALL be generated when the DeviceMuted attribute changes to True.</description>
+    </event>
+    <event side="server" code="0x07" name="MuteEnded" priority="info" optional="true">
+      <description>This event SHALL be generated when DeviceMuted attribute changes to False.</description>
+    </event>
+    <event side="server" code="0x08" name="InterconnectSmokeAlarm" priority="info" optional="true">
+      <description>This event SHALL be generated when the device hosting the cluster receives a remote smoke alarm.</description>
+    </event>
+    <event side="server" code="0x09" name="InterconnectCOAlarm" priority="info" optional="true">
+      <description>This event SHALL be generated when the device hosting the cluster receives a remote CO alarm.</description>
+    </event>
+    <event side="server" code="0x0A" name="AllClear" priority="info" optional="false">
+      <description>This event SHALL be generated when ExpressedState attribute returns to Normal state.</description>
+    </event>
+  </cluster>
+
+  <!-- Cluster data types -->
+  <enum name="AlarmStateEnum" type="ENUM8">
+    <cluster code="0x005C" />
+    <item value="0" name="Unknown" />
+    <item value="1" name="Normal" />
+    <item value="2" name="Warning" />
+    <item value="3" name="Critical" />
+  </enum>
+
+  <enum name="SensitivityEnum" type="ENUM8">
+    <cluster code="0x005C" />
+    <item value="0" name="High" />
+    <item value="1" name="Standard" />
+    <item value="2" name="Low" />
+  </enum>
+
+  <enum name="ExpressedStateEnum" type="ENUM8">
+    <cluster code="0x005C" />
+    <item value="0" name="Normal" />
+    <item value="1" name="SmokeAlarm" />
+    <item value="2" name="COAlarm" />
+    <item value="3" name="BatteryAlert" />
+    <item value="4" name="Testing" />
+    <item value="5" name="HardwareFault" />
+    <item value="6" name="EndOfService" />
+    <item value="7" name="InterconnectSmoke" />
+    <item value="8" name="InterconnectCO" />
+  </enum>
+
+  <enum name="MuteStateEnum" type="ENUM8">
+    <cluster code="0x005C" />
+    <item value="0" name="Unknown" />
+    <item value="1" name="NotMuted" />
+    <item value="2" name="Muted" />
+  </enum>
+
+  <enum name="EndOfServiceEnum" type="ENUM8">
+    <cluster code="0x005C" />
+    <item value="0" name="Unknown" />
+    <item value="1" name="Expired" />
+    <item value="2" name="Normal" />
+  </enum>
+
+  <enum name="ContaminationStateEnum" type="ENUM8">
+    <cluster code="0x005C" />
+    <item value="0" name="Unknown" />
+    <item value="1" name="Normal" />
+    <item value="2" name="Low" />
+    <item value="3" name="Warning" />
+    <item value="4" name="Critical" />
+  </enum>
+
+  <bitmap name="SmokeCOFeature" type="BITMAP32">
+    <cluster code="0x005C" />
+    <field name="SMOKE" mask="0x01" />
+    <field name="CO" mask="0x02" />
```

Comment:
```suggestion
    <field name="Smoke Alarm" mask="0x01" />
    <field name="CO alarm" mask="0x02" />
```
to match the spec, but note https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/7004